Explorer/Assets/DCL/Passport/Prefabs/Passport.prefab

```diff
@@ -4623,6 +4622,94 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: eddc1fbd4bd796345832996a9560b9bd, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!1 &7640579446494140225
+GameObject:
+  m_ObjectHideFlags: 0
```

Comment:
will it make sense to extract it to a prefab, so in case of future change it won't touch Passport prefab, but just its nested part?